### ISSUE TITLE
Создать концептуальную схему данных предметной области

### ISSUE THREAD

**Issue by Landaeva** [Landaeva/Kursach#7]

**Comment by Landaeva** [comment]
![курсачч](https://github.com/user-attachments/assets/df0dec14-35cc-4c71-b8c1-0af2c8ea4be8)

**Comment by wldmr-key** [comment]
тебя не смущает, что концептуальная схема очень слабо соответствует описанию предметной области? Особенно в части сущностей и связй

**Comment by Landaeva** [comment]
![image](https://github.com/user-attachments/assets/47a397c7-a715-4ec8-a681-9aa4dbb01146)

**Comment by Landaeva** [comment]
![image](https://github.com/user-attachments/assets/43fd899d-59df-404d-a0a1-0e93c2882ce5)

**Comment by Landaeva** [comment]
![image](https://github.com/user-attachments/assets/30ccd932-f920-4e99-872a-f845c0e21bb1)

**Comment by wldmr-key** [comment]
не очень вяжется с описанием предметной области. Давай, разберем на консультации

**Comment by Landaeva** [comment]
![image](https://github.com/user-attachments/assets/21f1e5d7-2698-4eb6-affd-b18cf729ea68)

**Comment by Landaeva** [comment]
![image](https://github.com/user-attachments/assets/e4d8bf2d-1421-4aa2-9fae-d0a127499bc1)

**Comment by Landaeva** [comment]
![image](https://github.com/user-attachments/assets/62c7044a-87b0-4e4a-9bd3-b12eee8583c8)

**Comment by wldmr-key** [comment]
насколько я помню мы на консультации эту схему разобрали...

**Comment by Landaeva** [comment]
Да, разобрали, там просто корректировка была насчет количества авто

**Comment by wldmr-key** [comment]
да и про тест-драйв вопросы были - кто записан на тест-драйв?

**Comment by Landaeva** [comment]
Записываем клиента, там же есть эта связь. Или нужно что-то еще?

**Comment by wldmr-key** [comment]
> Записываем клиента, там же есть эта связь. Или нужно что-то еще?

а-а-а, сорри, не на ту схему смотрел ((
